db/seeds/initial_seed.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
```

Comment:
What about just calling the file production.rb? Generally, I wish we had environment specific seeds rather than having the environment checks in the files.

Reply:
Is there anywhere we don't want this to run?

Reply:
No, if it's good enough for prod then it's good enough for every other env. What I feel queasy about is that we are running dummy structures as well and that has env checks in it. I'd sleep better if we had an env specific file that only has what we need to run in prod.

Reply:
I guess we could move the environment check out of the dummy structures file and into the main seed file?

Reply:
Happy with this is @amazimbe is

Reply:
Which file will we run for prod then?

Reply:
We run `db:seed` everywhere, the environment determines what runs